DoSomething/gladiator
# Update waiting rooms crud

#### What's this PR do?
* Removed the ability to create and destroy waiting rooms since these get created automatically when a contest gets created, and we also don't want to be deleting waiting rooms since a contest should never not have a waiting room. 

* Updated index and show views to work with the new waiting rooms table. 

* Updated the edit form and validation rules to work with the new waiting rooms table. 

#### How should this be manually tested?
Pull down this branch and check the CRUD functionality or should I say the RU functionality :smirk: 

Make sure it still works with the new DB structure. 

#### Any background context you want to provide?

Making these updates after #45 was done.

#### What are the relevant tickets?
Fixes #60 

@angaither @weerd @deadlybutter
